src/main/java/racingcar/race/RacingRegistry.java

```diff
@@ -0,0 +1,27 @@
+package racingcar.race;
+
+import racingcar.util.CommonUtil;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class RacingRegistry {
+    private static final String SPLIT_REGEX = ",";
+
+    private RacingRegistry() {
+
+    }
+
+    public static List<Car> getCarList(String nameString) {
+        CommonUtil.checkNullEmpty(nameString);
+
+        List<Car> carListJoinRace = new ArrayList<>();
+        String[] splitCars = nameString.split(SPLIT_REGEX);
+
+        Arrays.stream(splitCars).forEach(carName ->
+                carListJoinRace.add(new Car(carName.trim())));
+        return carListJoinRace;
```

Comment:
```suggestion
        String[] splitCars = nameString.split(SPLIT_REGEX);

        return Arrays.stream(splitCars)
                   .map(carName -> new Car(carName.trim()))
                   .collect(Collectors.toList());
```
이렇게 정리할 수 있을 거 같네요:)